crates/btc-notify/src/config.rs

```diff
@@ -14,7 +14,7 @@ use crate::constants::DEFAULT_BURY_DEPTH;
 #[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
 pub struct BtcZmqConfig {
     /// Depth at which a transaction is considered buried, defaults to [`DEFAULT_BURY_DEPTH`].
-    pub(crate) bury_depth: usize,
+    pub bury_depth: usize,
```

Comment:
I much prefer a getter than making this `pub` if not all other fields are `pub`.